### ISSUE TITLE
Push Notification can't activate

### ISSUE THREAD

**Issue by azplanlos** [thedillonb/CodeHub#121]
Push notifications are not working because they are disabled in control center. There is no way to activate as rights settings are not available.

![](http://i.imgur.com/jBndY40.jpg)

**Comment by thedillonb** [comment]
Hello @azplanlos, the image that you posted, how did you get there?

Are you going Settings -> Notifications -> CodeHub?

![image](https://cloud.githubusercontent.com/assets/1644068/6882336/109ba91a-d557-11e4-9b34-c39011ae63f4.png)

That's what I see when I go there. You should then be able to enable notifications.

**Comment by azplanlos** [comment]
Yes, it is code hubs options in the settings. But I don't get all this options

Von meinem iPhone gesendet

Am 28 Mar 2015 um 20:31 schrieb Dillon Buchanan <notifications@github.com<mailto:notifications@github.com>>:


Hello @azplanlos<https://github.com/azplanlos>, the image that you posted, how did you get there?

Are you going Settings -> Notifications -> CodeHub?

[image]<https://cloud.githubusercontent.com/assets/1644068/6882336/109ba91a-d557-11e4-9b34-c39011ae63f4.png>

That's what I see when I go there. You should then be able to enable notifications.

-

**Comment by thedillonb** [comment]
Interesting. I'll poke around and see if I can find out what's going on.

**Comment by azplanlos** [comment]
![](http://i.imgur.com/BoYKpnw.jpg)

Reinstalled the App but didn't solve the issue

**Comment by azplanlos** [comment]
By the way: same issue also on my iPad (both iOS 8.2).

**Comment by thedillonb** [comment]
Hey @azplanlos, just wanted to let you know I'm still looking into this. It's so weird that, even after a re-install, the app is unable to register for push notifications. I'm having trouble figuring out why iOS wouldn't be giving CodeHub a push notifications token. Until I resolve this, you are by all means welcome to ask for a refund from Apple for this.

**Comment by thedillonb** [comment]
Unreal, so I just figured it out. Apparently after I recompiled the application for iOS 8 there was a warning about a **deprecated** function. What Apple really means is that they introduced a breaking change since that function to register the device with APN doesn't actually work in iOS8.

`registerForRemoteNotificationTypes: is not supported in iOS 8.0 and later.`

So those who are trying to register cant. Those who already have, it still works.

I'm going to fix this as fast as my little fingers can type and push a new version to the app store.
